website/docs/reference/global-configs/version-compatibility.md

```diff
@@ -14,7 +14,7 @@ Running with dbt=1.0.0
 Found 13 models, 2 tests, 1 archives, 0 analyses, 204 macros, 2 operations....
 ```
 
-:::info dbt Cloud release tracks
+:::note dbt Cloud release tracks
 <Snippet path="_config-dbt-version-check" />
```

Comment:
Hiya @mirnawong1 

This is more of a question / suggestion. Just thinking of customers who may be visually impaired (this wasn't part of your PR btw) - is it possible to make the background a different colour as the text is quite hard to read.

Kind Regards
Natalie

Reply:
yea this callout does need tweaking -- currently this is how it looks like in dark mode and i changed the callout from info to note so it's at least visible. but the component actually needs updating so the expandable color changes in dark mode:

![Screenshot 2024-12-23 at 15 36 47](https://github.com/user-attachments/assets/391a5cf9-15dc-499e-9183-974e2ea3fe5a)